bitnami/rabbitmq/templates/manager-ap.yaml

```
@@ -0,0 +1,54 @@
+{{- if .Values.manager.privatelyExposed.enabled -}}
+apiVersion: security.istio.io/v1
+kind: AuthorizationPolicy
+metadata:
+  name: {{ .Release.Name }}-manager
```

The AP isn't specific to the manager, but is global to the whole rmq pod (cf the selector and the rules), so:
```suggestion
  name: {{ .Release.Name }}
```